server/handlers/cdn-caching.js

```diff
@@ -1,57 +1,114 @@
-const _ = require("lodash");
+const _ = require('lodash');
 
 exports.addCacheHeadersToResult = function addCacheHeadersToResult(
   res,
   cacheKeys,
-  cdnProvider = "cloudflare"
+  cdnProvider = 'cloudflare'
 ) {
   if (cacheKeys) {
-    if (cacheKeys === "DO_NOT_CACHE") {
-      res.setHeader("Cache-Control", "private,no-cache,no-store,max-age=0");
-      cdnProvider === "akamai" &&
-        res.setHeader("Edge-Control", "private,no-cache,no-store,max-age=0");
-      res.setHeader("Vary", "Accept-Encoding");
-      res.setHeader("Surrogate-Control", "private,no-cache,no-store,max-age=0");
+    if (cacheKeys === 'DO_NOT_CACHE') {
+      res.setHeader('Cache-Control', 'private,no-cache,no-store,max-age=0');
+      cdnProvider === 'akamai' &&
+        res.setHeader('Edge-Control', 'private,no-cache,no-store,max-age=0');
+      res.setHeader('Vary', 'Accept-Encoding');
+      res.setHeader('Surrogate-Control', 'private,no-cache,no-store,max-age=0');
+      res.setHeader(
+        'Content-Security-Policy',
+        `default-src data: 'unsafe-inline' 'unsafe-eval' https: http:;` +
```

Comment:
I'm not sure about 'unsafe-inline' or 'self'  here,

Reply:
ram told me to add this